cmd/template/cluster/provider/capg.go

```diff
@@ -182,6 +182,11 @@ func templateDefaultAppsGCP(ctx context.Context, k8sClient k8sclient.Interface,
 			Namespace:               organizationNamespace(config.Organization),
 			Version:                 appVersion,
 			UserConfigConfigMapName: configMapName,
+			DefaultingEnabled:       false,
+			UseClusterValuesConfig:  true,
+			ExtraLabels: map[string]string{
+				k8smetadata.ManagedBy: "cluster",
```

Comment:
This label is required so that the `app-admission-controller` doesn't block the `App` CR creation due to the `cluster-values` configmap not existing yet.